iOS/Views/MovieTableViewCell.swift

```diff
@@ -0,0 +1,42 @@
+import UIKit
+import Shared
+
+class MovieTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var movieBackdrop: UIImageView!
+    @IBOutlet weak var movieGenresImages: UIStackView!
```

Comment:
The pluralize var name is just because it is a Stack View. But I had the same question when I wrote :/